airbyte-webapp/src/views/Connection/ConnectionForm/components/SchemaChangeBackdrop.module.scss

```diff
@@ -0,0 +1,46 @@
+@use "scss/colors";
+@use "scss/variables";
+@use "scss/_z-indices";
+
+%cover {
+  width: 100%;
+  height: 100%;
+}
+
+.schemaChangeBackdropContainer {
+  @extend %cover;
+
+  position: relative;
+
+  .backdrop {
+    @extend %cover;
+
+    position: absolute;
+    background: linear-gradient(180deg, rgba(colors.$grey-50, 0.5) 0%, colors.$white 92.54%);
```

Comment:
Figma shows the old app bg color fading into white.  I've kept the gradient the same, and subbed in our new app bg color.